core/sources/domains.move

```diff
@@ -403,18 +403,38 @@ module aptos_names::domains {
         register_name_internal(sign, subdomain_name, domain_name, registration_duration_secs, 0);
     }
 
+    public entry fun force_set_expiration(
```

Comment:
Is this going to be done for 1 domain at a time? Might be good to provide a batch version here as well